src/editors/containers/ProblemEditor/components/EditProblemView/SettingsWidget/settingsComponents/ShowAnswerCard.jsx

```diff
@@ -1,9 +1,11 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
```

Comment:
This is something we should discuss more as a group first but I think we should move away from using connect and move towards using the useSelector hook. The studioEndpointUrl could be pulled from state through useSelector(selectors.app.studioEndpointUrl).